include/twist_mux/twist_mux.h

```diff
@@ -82,7 +83,9 @@ class TwistMux
   boost::shared_ptr<velocity_topic_container> velocity_hs_;
   boost::shared_ptr<lock_topic_container>     lock_hs_;
 
-  ros::Publisher cmd_pub_;
+  ros::Publisher cmd_pub_, status_pub_;
```

Comment:
wouldn't it be handier to add a new message that contains these two strings and publish that?

Reply:
I usually try to avoid adding custom messages for stuff like this. Unless you really believe that a new message type is the best way to go and don't like the current implementation of two topics, I'm thinking it might be better to either concatenate the strings with an easily parse-able delimiter or potentially use something like the diagnostics messages.

Reply:
I made the other couple small changes, let me know what you think the best course of action is on this. Unfortunately  I no longer have a melodic setup with ROS1 so I may not be able to make + test any major modifications anytime soon.